lib/travis/conditions/v1/data.rb

```diff
@@ -13,7 +13,9 @@ def [](key)
         end
 
         def env(key)
-          data.fetch(:env, {})[key.to_sym]
+          value = data.fetch(:env, {})[key.to_sym]
+          value = value.gsub(/(^["']|["']$)/, '') if value.respond_to?(:gsub)
```

Comment:
Not sure if this is entirely correct (though it would probably work most of the time). A contorted example might be `foo\"`, an unbalanced quotation mark but escaped and valid at the end of a string. Also, this does not necessarily match the same quotation marks `'foo"`; can't think of a situation where such a string is meaningful, but you never know!

We'd want something like

```
value.gsub(/^(["'])([^"']*)\1$/,'\2')
```